app/script/conversation/ConversationMapper.js

```diff
@@ -285,36 +288,77 @@ z.conversation.ConversationMapper = class ConversationMapper {
     return conversationEntity;
   }
 
+  /**
+   * Get the valid notification state.
+   *
+   * @param {z.conversation.NotificationSetting.STATE} [notificationState] - Bit mask based notification setting
+   * @param {boolean} [deprecatedMutedState] - Outdated muted state
+   * @returns {z.conversation.NotificationSetting.STATE} validated notification setting
+   */
+  getNotificationState(notificationState, deprecatedMutedState) {
+    if (notificationState === undefined || notificationState === null) {
```

Comment:
Instead of checking for `undefined` or `null` we can maybe check that the `notificationState` given is in the enum `z.conversation.NotificationSetting.STATE`